tests/visual/treemap.html

```diff
@@ -0,0 +1,140 @@
+<!doctype html>
+<html lang="en" class="k-typography k-no-animations">
+<head>
+    <title>Chart</title>
+    <meta charset="utf-8" />
+    <link rel="stylesheet" href="../../packages/default/dist/all.css" />
+    <link rel="stylesheet" href="assets/styles.css" />
+    <script src="assets/scripts.js"></script>
+
+    <script src="https://kendo.cdn.telerik.com/2019.1.115/js/jquery.min.js"></script>
+    <script src="https://kendo.cdn.telerik.com/2019.1.115/js/kendo.all.min.js"></script>
```

Comment:
Please remove those -- they are not needed for the test

Reply:
These should be removed now